### PR TITLE
Fix/tao 4963 end of test connectivity error

### PR DESCRIPTION
**Pull request by eacraft13**
Bugfix - ACT
Jira: https://oat-sa.atlassian.net/browse/TAO-4963

To test, you only need to click 'end test' on an item while disconnected.

### CODE REVIEW

File: views/js/runner/plugins/controls/connectivity/connectivity.js [hunk 1]
```diff
@@ -84,7 +84,10 @@ define([
                 }
 
                 //offline navigation error, we pause the test
-                if(err.source === 'navigator' && err.purpose === 'proxy' && err.code === 404){
+                if (
+                    proxy.isOffline() ||
+                    (err.source === 'navigator' && err.purpose === 'proxy' && err.code === 404)
```
Comment (dietertaotesting) [inline views/js/runner/plugins/controls/connectivity/connectivity.js]: I don't really see how this could be improved, I'd leave it like that

Comment (eacraft13) [reply views/js/runner/plugins/controls/connectivity/connectivity.js]: Ok, thanks for looking @dietertaotesting. I'm going to try to give it one last review today and I'll probably assign it to you tomorrow.

**Comment by eacraft13** [comment]
@dietertaotesting thanks again for looking. I agree, checking if the proxy is offline is a more robust way of handling the error.

**Comment by eacraft13** [comment]
@dietertaotesting, I'm unable to get the reference error ('content' of null) on my branch. Perhaps the js was not loaded (maybe via caching of js from the develop branch)? If you have time, could you walk me through how to produce that error again?

**Pull request merged**